### PR TITLE
FEATURE FREEZE FEBRUARY 

### PR DESCRIPTION
**Pull request by MrMelbert**
### There will be a feature freeze for the month of February. We're approaching 1,800 issues (2 away at the time of writing) and have a ton of runtimes going on during the rounds, and would like to whittle them down.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## /tg/station codebase
+## my codebase
```
Comment (JohnFulpWillard) [inline README.md:1]:
```suggestion
## JohnFulpWillard's codebase
```

Comment (Iamgoofball) [reply README.md:1]:
```suggestion
## our codebase
```

Comment (JohnFulpWillard) [reply README.md:1]: yeah i prefer goof's solution

**Comment by Mothblocks** [comment]
i'll hopefully get https://runtimes.moth.fans/ fixed by then

**Comment by Wallemations** [comment]
I forgot do sprites count as features or just things with the feature tag

**Comment by ArcaneMusic** [comment]
Features are anything that mechanically adds content to the game.

Making a new type of plant, seeds, and produce? That's a feature.

Adding new sprites for an existing type of plant? That's just sprites. That's fine.

Refactoring a type of plant to behave better  without changing the functionality? Also fine.

Changing the balance of a plant to deal more damage/interact with another system? That's a balance PR. Not fine.

Big golden rule is, if it's adding content to the game, it's probably better to wait. But feel free to ask if you're not sure, especially if it could go either way.

**Pull request closed without merging**